src/test/java/uk/gov/hmcts/reform/professionalapi/service/impl/ProfessionalUserServiceImplTest.java

```diff
@@ -233,69 +233,41 @@ public void findUsersByOrganisation_with_status_active() throws Exception {
 
         ResponseEntity responseEntity = professionalUserService.findProfessionalUsersByOrganisation(organisation, "false", true, "Active");
 
+        ProfessionalUsersEntityResponse professionalUsersEntityResponse1 = ((ProfessionalUsersEntityResponse)responseEntity.getBody());
+
         Mockito.verify(professionalUserRepository, Mockito.times(1)).findByOrganisation(organisation);
```

Comment:
could you please move Mockito to package level

Reply:
done